compiler/control/OMROptions.cpp

```diff
@@ -1521,8 +1521,11 @@ OMR::Options::setRegex(char *option, void *base, TR::OptionTable *entry)
    {
    TR::SimpleRegex * regex = TR::SimpleRegex::create(option);
    *((TR::SimpleRegex**)((char*)base+entry->parm1)) = regex;
-   if (!regex)
+   if (!regex) {
```

Comment:
To remain consistent with the existing coding style, we need to format braces the following way:
```c++
if(condition)
   {
   //body
   }
```

Reply:
agreed

Reply:
This could be a writelinelocked